### PR TITLE
New version: h3poteto.whalebird-desktop version 4.7.3

### PR DESCRIPTION
**Pull request by h3poteto**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91733)

### CODE REVIEW

**Review by OverRevvv — APPROVED** [review]

File: manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2022-12-18
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml:17]:
```suggestion
- Architecture: x64
```

File: manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  Scope: user
+  InstallerUrl: https://github.com/h3poteto/whalebird-desktop/releases/download/4.7.3/Whalebird-4.7.3-windows-x64.exe
+  InstallerSha256: 4BD577109427A2340B899EA2190D553FC36F70F7D90C3CA959CF838B51C29F21
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml:23]:
```suggestion
- Architecture: x64
```

File: manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  Scope: machine
+  InstallerUrl: https://github.com/h3poteto/whalebird-desktop/releases/download/4.7.3/Whalebird-4.7.3-windows-x64.exe
+  InstallerSha256: 4BD577109427A2340B899EA2190D553FC36F70F7D90C3CA959CF838B51C29F21
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml:29]:
```suggestion
- Architecture: ia32
```

Comment (stephengillie) [reply manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml:29]: I'm not sure if Winget supports ia32, or if that architecture has a different internal name. This section, and the following section, might need to be removed to pass validation.

File: manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  Scope: user
+  InstallerUrl: https://github.com/h3poteto/whalebird-desktop/releases/download/4.7.3/Whalebird-4.7.3-windows-ia32.exe
+  InstallerSha256: 9AEF00BBFDA7A757DFA14E8BAE2335BAD47E197824F16C38E462B6101DA0EB1F
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml:35]:
```suggestion
- Architecture: ia32
```

Comment (stephengillie) [reply manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml:35]: As above, so below.

File: manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml (hunk 3 above)
Comment (Trenly) [reply manifests/h/h3poteto/whalebird-desktop/4.7.3/h3poteto.whalebird-desktop.installer.yaml:29]: > I'm not sure if Winget supports ia32, or if that architecture has a different internal name. This section, and the following section, might need to be removed to pass validation.

ia32 isn't part of the enum for supported architectures; x86 is the correct architecture to use for 32 bit support

**Pull request closed without merging**